app/controllers/carts_controller.rb

```diff
@@ -19,6 +18,12 @@ def check_out
     redirect_to new_order_path
   end
 
+  # TODO: V2 check_out_carts_path
+  # Turbo stream needs a redirect
+  # def check_out
+  #   redirect_to new_order_path
+  # end
+
```

Comment:
I Was thinking about adopting different payment methods apart from Stripe, so puts this here as a reminder.